### ISSUE TITLE
Expand Export to MBOX

### ISSUE THREAD

**Issue by peterchanws** [ePADD/epadd#221]
This is part of #196 
Remove "Export to MBOX" and add New Section: Export Messages to MBOX

Drop down box contain:
All messages to MBOX file
All non-restricted messages to MBOX file
All restricted messages to MBOX file

User click "Export" to start downloading the selected file.

**Comment by peterchanws** [comment]
@chinuhub 

all messages with labels (any- general/restricted) is not equal to any of the options I mentioned.

All messages - users can do a blank search, but it may take a long time
All non-restricted messages - there is no way users can use advanced search for this set
All restricted messages - users can select all restriction labels, however, it will be cleaner to do in the way I suggested

**Comment by hangal** [comment]
yes, I think it's reasonable to have export to mbox for these options.

**Comment by chinuhub** [comment]
@peterchanws For the definition of restricted messages; is it the same as we use to decide if a message should be exported to the next module or not? Essentially a message is not restricted iff
1. It has cleared for release  (even if it has a general restriction label too).
2. A message with time restriction label that has expired 
3. A message with only general labels or no labels 

Everything else is a restricted message,

**Comment by peterchanws** [comment]
@chinuhub 

For the definition of restricted messages; is it the same as we use to decide if a message should be exported to the next module or not? YES

A message is not restricted if
1. It has cleared for release (even if it has a general restriction label too).
2. A message with only general labels or no labels
Everything else is a restricted message.

A message with time restriction label that has expired but without cleared for release label is still a restricted message.

**Comment by peterchanws** [comment]
@chinuhub 
Could you separate Export messages in mbox section out?

**Comment by peterchanws** [comment]
epadd-standalone-v6-beta-18June.jar
works as requested

**Comment by peterchanws** [comment]
The non restricted messages exported contain restricted messages - messages with time restriction label that has expired but without cleared for release label

**Comment by peterchanws** [comment]
epadd-standalone-v6-beta-21June.jar
Took 19:19 min to export all messages to mbox
